src/sources/kubernetes_logs/k8s_paths_provider.rs

```diff
@@ -45,7 +45,7 @@ impl PathsProvider for K8sPathsProvider {
             // filter out pods where we haven't fetched the namespace metadata yet
             // they will be picked up on a later run
             .filter(|pod| {
-                trace!(message = "Verifying Namespace metadata for pod.", pod = ?pod);
+                trace!(message = "Verifying Namespace metadata for pod.", pod = ?pod.metadata.name);
```

Comment:
This, and the `trace!` below would print the _entire_ pod resource, which is a rather enormous struct - this just prints `Some("<podname>")`.